### PR TITLE
Replace instrumentation module/package instances with instrumentation library

### PR DESCRIPTION
**Pull request by svrnm**
This PR replaces some instances where "(auto) instrumentation module" or "(auto) instrumentation package" can be replaced with "instrumentation library"

### CODE REVIEW

**Review by tsloughter — APPROVED** [review]

File: content/en/blog/2022/dotnet-instrumentation-first-beta.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ of the
 [OpenTelemetry .NET Automatic Instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation)
 project!
 
-Without this project, .NET developers need to use instrumentation packages to
+Without this project, .NET developers need to use instrumentation libraries to
 automatically generate telemetry data. For example, to instrument inbound
-ASP.NET Core requests, you need to use the ASP.NET Core instrumentation package
+ASP.NET Core requests, you need to use the ASP.NET Core instrumentation libraries
```
Comment (Kielek) [inline content/en/blog/2022/dotnet-instrumentation-first-beta.md:17]:
```suggestion
ASP.NET Core requests, you need to use the ASP.NET Core instrumentation library
```

**Review by pichlermarc — APPROVED** [review]

File: content/en/docs/languages/net/libraries.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ dotnet add package OpenTelemetry.Extensions.Hosting
 dotnet add package OpenTelemetry.Exporter.Console
 ```
 
-Then you can install the Instrumentation packages
+Then you can install the instrumentation libraries
```
Comment (tiffany76) [inline content/en/docs/languages/net/libraries.md:54]:
```suggestion
Then you can install the instrumentation libraries:
```

File: content/en/docs/languages/php/getting-started.md [hunk 3]
```diff
@@ -142,7 +142,7 @@ Next, you’ll use the OpenTelemetry PHP extension to
      open-telemetry/opentelemetry-auto-slim
    ```
 
-With the OpenTelemetry PHP extension set up and an auto-instrumentation package
+With the OpenTelemetry PHP extension set up and a instrumentation library
```
Comment (tiffany76) [inline content/en/docs/languages/php/getting-started.md:145]:
```suggestion
With the OpenTelemetry PHP extension set up and an instrumentation library
```

**Review by tiffany76 — APPROVED** [review]

**Review by theletterf — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

**Comment by theletterf** [comment]
@svrnm Looks like we only need @open-telemetry/php-approvers approval.

**Pull request merged**